src/screens/About/Team/Team.style.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { Paragraph } from 'components/Markdown';
+import Grid from '@material-ui/core/Grid';
+import { COLOR_MAP } from 'common/colors';
+
+export const Headshot = styled.img`
+  border-radius: 50%;
+  max-width: 4rem;
+  max-height: 4rem;
```

Comment:
css question, should rem units be used for widths over pixels?

Reply:
hmm my very incomplete understanding of the use of rems vs. pixels is that rems are better for accessibility. but tbd